tools/generate-helpers/pg-table-bindings/list.go

```diff
@@ -30,7 +30,9 @@ func init() {
 		&storage.ComplianceRunMetadata{}:                        resources.Compliance,
 		&storage.ComplianceRunResults{}:                         resources.Compliance,
 		&storage.ExternalBackup{}:                               resources.BackupPlugins,
+		&storage.ImageComponent{}:                               resources.Image,
```

Comment:
This should be still the old, to-be-deprecated resources.

This will allow existing clients to use the old resource as well as the new resource. Only specifying `resource.Image/resource.Node` here will lead to the old resource as not being valid, since `resources.Image/Node` doesn't know about resource it replaces.

Reply:
The postgres stores were added after 3.70. IMHO, they should directly be plugged on the new resource.

Reply:
Fine by me!